content/account_management/settings.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Settings
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+---
+
+{{< img src="account_management/settings/settings_preferences.png" alt="Settings Preference" >}}
+
+This panel allow you: 
+
+* To chose your timezone
+* Enable Desktop notifications
+* Custom your Email subscription parameters.
```

Comment:
"Customize your email subscription settings"

Why does only one line have a period? We should be consistent.

Reply:
👍